packages/zoe/src/contractFacet/seat.js

```diff
@@ -41,6 +42,7 @@ export const makeZcfSeatAdminKit = (
       assertExitedFalse();
       exited = true;
     },
+    updateProposal: newProposal => (proposal = newProposal),
```

Comment:
Hmm, I really do not like the idea that the proposal is updated. I think we should only have one path for adding a proposal to a seat, and the old way of making an invitation should use the new path under the hood.